README.md

```diff
@@ -1,14 +1,15 @@
 ## Broadbean aka QCoDeS Pulse Builder
 
 [ ![PyPi](https://badge.fury.io/py/broadbean.svg) ](https://badge.fury.io/py/broadbean)
-[ ![DOCS](https://readthedocs.org/projects/broadbean/badge/?version=latest) ](http://broadbean.readthedocs.io/en/latest/?badge=latest)
+[ ![Build Status Github Docs](https://github.com/QCoDeS/broadbean/workflows/build%20docs/badge.svg) ](https://github.com/QCoDeS/broadbean/actions?query=workflow%3A%22build+docs%22)
+[ ![DOCS](https://img.shields.io/badge/read%20-thedocs-ff66b4.svg) ](https://qcodes.github.io/broadbean/index.html#)
 [ ![PyPI python versions](https://img.shields.io/pypi/pyversions/broadbean.svg) ](https://pypi.python.org/pypi/broadbean/)
 [ ![Build Status Github](https://github.com/QCoDeS/broadbean/workflows/Run%20mypy%20and%20pytest/badge.svg) ](https://github.com/QCoDeS/broadbean/actions?query=workflow%3A%22Run+mypy+and+pytest%22)
 
 A library for making pulses that can be leveraged with QCoDeS (in
 particular its Tektronix AWG 5014 driver), but also works as standalone.
```

Comment:
Should this be updated to also mention the newer Tektronix drivers?

Reply:
Yes, we definitely should but I wanted to confirm everything is working well with the newer driver before updating the readme.